routes/poll.js

```diff
@@ -0,0 +1,73 @@
+var express = require('express');
+var config = require('../config.json');
+var UserModel = require('../models/user');
+var QuestionModel = require('../models/question');
+var AnswerModel = require('../models/answer');
+var router = express.Router();
+
+
+/* GET create page. */
+router.get('/create-poll', function(req, res) {
+  res.render('createPoll', { title: 'VibeCheck' });
+});
+
+router.post('/create-poll', function(req, res) {
+  var newQuestion= new QuestionModel();
+  
+  console.log(req.body.question);
+  console.log(req.body.choice1);
+  console.log(req.body.choice2);
+  console.log(req.body.choice3);
+  console.log(req.body.choice4);
+  console.log(req.body.choice5);
+  console.log(newQuestion.text);
+  console.log(req.user.username);
+  //mongoose.Types.ObjectId(req.user.userId)
+  newQuestion.text = req.body.question;
+  newQuestion.postedBy = "5fc581f2131e949d944b5a81";
```

Comment:
did you try `newQuestion.postedBy = mongoose.Types.ObjectId(req.user.userId);` ?

Reply:
If my suggestion above doesnt work, you are going to have to query the database for the user with `req.user.username`